packages/plugins/response-cache/src/index.ts

```diff
@@ -170,10 +170,14 @@ export function useResponseCache(options: UseResponseCacheParameter): Plugin {
       if (enabled(request)) {
         const cachedResponse = await cache.get(operationId);
         if (cachedResponse) {
+          const responseWithSymbol = {
+            ...cachedResponse,
+            [Symbol.for('servedFromResponseCache')]: true,
```

Comment:
So finaly you don't expose the entire metadata object ?

Reply:
it would be `{ hit: true }` always in this scenario anyways, we can change it to an object later on if there is more information to store.